### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by vinchevskaya**

### CODE REVIEW

File: Beginning.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (IgorKheidorov) [inline Beginning.cs]: remiove unnecessary usings

File: Beginning.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DEV_1
+{
+
```
Comment (IgorKheidorov) [inline Beginning.cs]: Code style, empty line

File: Beginning.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DEV_1
+{
+
+    class Beginning
```
Comment (IgorKheidorov) [inline Beginning.cs]: Funny class name, explains nothing )

File: Workwithstring.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            return maxLength;
+        }
+
+        public Workwithstring(string ourString)
+        {
+            currentString = ourString;
+        }
+    }
+}
+
```
Comment (IgorKheidorov) [inline Workwithstring.cs]: empty lines

File: Workwithstring.cs [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+                }
+                else
+                {
+                    firstSymbol = currentString[i];
+                    count = 0;
+                }
+            }
+            return maxLength;
+        }
+
```
Comment (IgorKheidorov) [inline Workwithstring.cs]: Constructor has to first

File: Workwithstring.cs [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    /// </summary>
+    class Workwithstring
+    {
+        private string currentString;
+
+        /// <summary>
+        /// This method finds repetitions of the symbols in the string
+        /// </summary>
+        /// <returns> The maximum number of symbol repetitions per line </returns>
+        public int maxRepetition()
```
Comment (IgorKheidorov) [inline Workwithstring.cs]: Code style violation, UpperCase
At least MaxRepetions

File: Workwithstring.cs [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DEV_1
+{
+    /// <summary>
+    /// This class is for working with a string
+    /// </summary>
```
Comment (IgorKheidorov) [inline Workwithstring.cs]: Code style violation, better at least WorkWithString

File: Beginning.cs [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+            {
+                Workwithstring String = new Workwithstring(symbols[0]);
+                Console.WriteLine(String.maxRepetition());
+            }
+            else
+            {
+                Console.WriteLine("wrong view");
+            }
+        }
+
```
Comment (IgorKheidorov) [inline Beginning.cs]: Empty strings

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: DEV_1.csproj.user [hunk 9]
```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (IgorKheidorov) [inline DEV_1.csproj.user:1]: Remove file from git, this is local  settings

File: WorkWithString.cs [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    {
+        public WorkWithString(string ourString)
+        {
+            currentString = ourString;
+        }
+        private string currentString;
+        /// <summary>
+        /// This method finds repetitions of the symbols in the string
+        /// </summary>
+        /// <returns> The maximum number of symbol repetitions per line </returns>
```
Comment (IgorKheidorov) [inline WorkWithString.cs:16]: Better GetMaxRepetitions

File: EntryPoint.cs [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+                Console.WriteLine(String.maxRepetition());
+            }
+            else
+            {
+                Console.WriteLine("wrong view");
+            }
+        }
+    }
+}
+
```
Comment (IgorKheidorov) [inline EntryPoint.cs:24]: Empty lines

File: EntryPoint.cs [hunk 12]
```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace DEV_1
+{
+    class Entrypoint
+    {
+        /// <summary>
+        /// This class deduced the number of characters from the longest repetition
```
Comment (IgorKheidorov) [inline EntryPoint.cs:8]: Check English grammar

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

**Review by AlexanderSoroka — APPROVED** [review]
Not accepted as some comments were just ignored

**Pull request merged**